html-templates/projects/projectDeleted.tpl

```diff
@@ -0,0 +1,7 @@
+{extends designs/site.tpl}
+
+{block title}Project Deleted &mdash; {$dwoo.parent}{/block}
+
+{block content}
+    <p>Project {$data->Title}</a> deleted.</p>       
```

Comment:
@bennovack the title should be piped through escape since there are no limits on characters used in Title: `{$data->Title|escape}`